### ISSUE TITLE
'pod install' causes Xcode to hang.

### ISSUE THREAD

**Issue by michael-snapptec** [CocoaPods/CocoaPods#6369]
# Report

## What did you do?

Run `pod install` or `pod update`

## What did you expect to happen?

Install all pod dependencies correctly.

## What happened instead?

Returning to Xcode shows that the app is hung and needs to be Force Quit. This seems to be a consistent issue. Sometimes, closing Xcode before running `pod update` prevents the hang. Sometimes, it still hangs on next launch.

### Stack

```
   CocoaPods : 1.1.1
        Ruby : ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin16]
    RubyGems : 2.5.1
        Host : Mac OS X 10.12.2 (16C67)
       Xcode : 8.2.1 (8C1002)
         Git : git version 2.10.1 (Apple Git-78)
Ruby lib dir : /Users/mike/.rbenv/versions/2.3.1/lib
Repositories : forcedotcom - https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Specs.git @ f58e547161d6bfc0199f9939e576dd7587cfcbb3
               master - https://github.com/CocoaPods/Specs.git @ 9b494e6ece34864e4028b8f436e00a58c2f4984e
```

### Installation Source

```
Executable Path: /Users/mike/.rbenv/versions/2.3.1/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.1.2
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
target 'v5snapp' do
    use_frameworks!
    
    pod 'Fabric'
    pod 'Crashlytics'
    pod 'Charts'
    pod 'GLCalendarView', '~> 1.0.0'
    pod 'GlyuckDataGrid'
    pod 'MBProgressHUD', '~> 0.9.2'
    pod 'RNCryptor-objc'
end

```

**Comment by benasher44** [comment]
Hi @michael-snapptec! Can you provide a sample project that reproduces the issue? Also, can you try running `pod deintegrate` and then `pod install` again to see if that helps? Thanks!

**Comment by skrew** [comment]
Have the same problem for months now, obj-c or swift projects.

I can't provide a sample project because it's not easy to reproduce each time. When i have this problem (like right now) I have to start, kill, start, kill (...) Xcode until it finally run.. It can take 2 minutes or 2 hours...

FYI, tested with `pod deintegrate` and `pod install` without success...

**Comment by benasher44** [comment]
@michael-snapptec if you reboot, does that fix the issue for you? This seems to have helped for @skrew. Thanks!

**Comment by skrew** [comment]
Got the problem again...
Closing Xcode before `pod install`or `pod update` are useless, the problem are stil here.
I had rebooted my Mac and the project can be loaded again.

**Comment by skrew** [comment]
Got the problem again... This time, rebooting won't fix the problem.

I can provide a TeamViewer session, because sending project won't help... 
Tell me if any cocoapod developer want to see what's happening, it's really boring...

**Comment by benasher44** [comment]
Oh great! Can you try removing the pod you added to see if that fixes the issue? Can you also try adding and removing other pods to see if you can narrow it down to 1? If you can, I can probably be of more help. Thanks!

**Comment by lucasilverentand** [comment]
I've found that running CleanMyMac 3 resolves the issue and I can open xcode again without any problems after that. Might be something going wrong in Xcode itself...

**Comment by skrew** [comment]
@benasher44: I will try next time, but I'd already tested to remove most recent pod installed and it don't fix the issue.

**Comment by benasher44** [comment]
@skrew thanks! I look forward to hearing how it goes!

**Comment by benasher44** [comment]
I'm going to close this one since there hasn't been any activity on it in some time, and it seems to be an issue with Xcode rather than CocoaPods. If you feel like you're still experiencing this, please file a new issue and fill out the template with information using the latest CocoaPods. Have a great day!

**Comment by iankchristie** [comment]
Hello,
I'm currently having this problem too :(
The project unexpectedly started throwing an error about not having a file to GoogleToolboxForMac, which I had been using without problems up until that point. So I did a '''pod install''' and not it hung on reopening.
I tried the ```pod deintegrate``` and then ```pod install``` to no avail. Then I tried a reboot. It looked like it almost helped since it loaded a little more information on xCode, but then hung again.
